test/evergreen.yml

```
@@ -5538,9 +5538,11 @@ buildvariants:
     ENABLE_TCMALLOC: -DENABLE_TCMALLOC=1
     num_jobs: $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
     cmake_generator: "Unix Makefiles"
+  # Using large distro for a few tests here as more memory is required. See WT-12378.
```

We try not to mention tickets in the code. One can use `git blame` to check where those changes come from!
```suggestion
  # Using large distro for a few tests here as more memory is required.
```